src/components/enterprise-page/data/hooks.js

```diff
@@ -88,4 +89,47 @@ export function useEnterpriseCustomerConfig(enterpriseSlug, useCache = true) {
   }, [enterpriseSlug]);
 
   return [enterpriseConfig, fetchError];
-}
+};
+
+/**
+ * @param {string} [enterpriseId] enterprise UUID
+ * @param {object} [user] user object containing JWT roles
+ */
```

Comment:
nit: could use a comment to the effect of "sets the user's active enterprise an API call and forces login_refresh to re-order the roles inside the user's JWT"